salt/cloud/clouds/dimensiondata.py

```diff
@@ -38,6 +38,7 @@
 
     HAS_LIBCLOUD = True
 except ImportError:
+    log.exception("You do not have apache-libcloud 0.20<= installed")
```

Comment:
This warning needs to be removed as `log` isn't available at the point yet:
```
[ERROR   ] Failed to import clouds dimensiondata, this is due most likely to a syntax error:
Traceback (most recent call last):
  File "/root/SaltStack/salt/salt/loader.py", line 1296, in _load_module
    ), fn_, fpath, desc)
  File "/root/SaltStack/salt/salt/cloud/clouds/dimensiondata.py", line 41, in <module>
    log.exception("You do not have apache-libcloud 0.20<= installed")
NameError: name 'log' is not defined
```
The warning that `apache-libcloud` is missing should already be handled by the `get_dependencies` check in the `__virtual__()` function.